bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckRun.java

```diff
@@ -596,31 +597,43 @@ private String warningListToText(List<String> additionalErrors) {
         return text;
     }
 
-    private Optional<String> getReviewersList() {
+    private Optional<String> getReviewersList(boolean tooFewReviewers) {
         var reviewers = activeReviews.stream()
-                               .filter(review -> review.verdict() == Review.Verdict.APPROVED)
-                               .map(review -> {
-                                   var entry = " * " + formatReviewer(review.reviewer());
-                                   if (!review.targetRef().equals(pr.targetRef())) {
-                                       entry += " 🔄 Re-review required (review was made when pull request targeted the [" + review.targetRef()
-                                               + "](" + pr.repository().webUrl(new Branch(review.targetRef())) + ") branch)";
-                                   } else {
-                                       var hash = review.hash();
-                                       if (hash.isPresent()) {
-                                           if (!reviewCoverage.covers(review)) {
-                                               entry += " 🔄 Re-review required (review applies to [" + hash.get().abbreviate()
-                                                       + "](" + pr.filesUrl(hash.get()) + "))";
-                                           } else {
-                                               entry += " ⚠️ Review applies to [" + hash.get().abbreviate()
-                                                       + "](" + pr.filesUrl(hash.get()) + ")";
-                                           }
-                                       } else {
-                                           entry += " 🔄 Re-review required (review applies to a commit that is no longer present)";
-                                       }
-                                   }
-                                   return entry;
-                               })
-                               .collect(Collectors.joining("\n"));
+                .filter(review -> review.verdict() == Review.Verdict.APPROVED)
+                .map(review -> {
+                    var entry = " * " + formatReviewer(review.reviewer());
+                    if (!review.targetRef().equals(pr.targetRef())) {
+                        if (useStaleReviews || tooFewReviewers) {
+                            entry += " 🔄 Re-review required (review was made when pull request targeted the [" + review.targetRef()
+                                    + "](" + pr.repository().webUrl(new Branch(review.targetRef())) + ") branch)";
```

Comment:
Why should we always display  "🔄" when `useStaleReviews==true`?

Reply:
I see, previously we always require re-review when target branch changes.
My question should be why don't we display "🔄" when `useStaleReviews==false`?